src/Explorer/Controls/CommandButton/CommandButtonComponent.tsx

```diff
@@ -31,7 +31,7 @@ export interface CommandButtonComponentProps {
   /**
    * Click handler for command button click
    */
-  onCommandClick: (e: React.SyntheticEvent | KeyboardEvent) => void;
+  onCommandClick?: (e: React.SyntheticEvent | KeyboardEvent) => void;
```

Comment:
What is the use case for a button with no `onCommandClick`?

Reply:
The "View" toolbar button used to show the Horizontal/Vertical options is just a menu root. Clicking "View" itself has no action. That's the scenario that motivated this change.